funnel/models/rsvp.py

```diff
@@ -78,8 +79,23 @@ def response(self):
         message=__("Your response has been saved"),
         type='primary',
     )
-    def rsvp_yes(self):
-        pass
+    def rsvp_yes(self, actor: User):
+        previous_membership = (
+            CommentsetMembership.query.filter(CommentsetMembership.is_active)
+            .filter_by(
+                commentset=self.project.commentset,
+                user_id=actor.id,
+            )
+            .one_or_none()
+        )
+        if previous_membership is None:
+            membership = CommentsetMembership(
+                commentset=self.project.commentset,
+                user=actor,
+                granted_by=actor,
+                is_subscriber=True,
+            )
+            db.session.add(membership)
```

Comment:
`previous_membership.replace()` is missing. Also, do we need that `is_subscriber` column? Couldn't `is_active` alone do it?